product/opni/components/dialogs/AddTokenDialog.vue

```diff
@@ -30,28 +34,80 @@ export default {
       {
         label: '7 days',
         value: '604800s'
+      },
+      {
+        label: 'custom',
+        value: 'custom'
```

Comment:
I saw you started doing form validation elsewhere. You may want to make sure this is within the expected bounds.